api/urls.py

```diff
@@ -126,6 +127,7 @@ def __init__(self, trailing_slash="/?"):
         name="sample-import-file",
     ),
     path("filter-values/", AvailableFilterValues.as_view(), name="available-filters"),
+    path("downloads/?P<filewanted>", Downloads.as_view(), name="Downloads"),
```

Comment:
we should have the query param checked inside the view. this will give us flexibility with the endpoint. see [this](https://www.django-rest-framework.org/api-guide/requests/)